src/Application/QueryStringParser.php

```diff
@@ -159,4 +159,13 @@ private function findOperator( string $constraint ): string {
 		return '';
 	}
 
+	/**
+	 * @return string[]
+	 */
+	private function getNonEmptyOrValues( string $value ): array {
+		return array_filter(
+			explode( '|', $value ),
+			fn( string $orValue ): bool => $orValue !== ''
```

Comment:
Not handling whitespace, since that depends on quote handling (#88), otherwise the search engine will just consider it a separate token.